### PR TITLE
ledger: change default folder to `.namada`

### PR DESCRIPTION
**Pull request by Fraccaman**
Lets not merge this until we have fixed our internal tools

### CODE REVIEW

File: apps/src/lib/cli.rs [hunk 1]
```diff
@@ -1272,7 +1272,7 @@ pub mod args {
     const ARCHIVE_DIR: ArgOpt<PathBuf> = arg_opt("archive-dir");
     const BASE_DIR: ArgDefault<PathBuf> = arg_default(
         "base-dir",
-        DefaultFn(|| match env::var("ANOMA_BASE_DIR") {
+        DefaultFn(|| match env::var("NAMADA_BASE_DIR") {
```
Comment (tzemanovic) [inline apps/src/lib/cli.rs:1275]: Let’s not change the env vars names

**Review by bengtlofgren — COMMENTED** [review]
With the help of Tomas ofc

**Comment by tzemanovic** [comment]
I think we can maybe close this and apply all the breaking changes at once in #794

**Comment by tzemanovic** [comment]
closed in favor of https://github.com/anoma/namada/pull/794

**Pull request closed without merging**